one_fm/one_fm/doctype/attendance_check/attendance_check.py

```diff
@@ -301,10 +301,16 @@ def create_attendance_check(attendance_date=None):
         if not attendance_date:
             attendance_date = add_days(today(), -1)
 
+        employee_with_shift_permission = [i.employee for i in frappe.db.get_list('Shift Permission', {
```

Comment:
you could use the frappe.db.get_list("Shift Permission", filter=filter, pluck="employee), please check this

Reply:
Noted